<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectDialog</class>
 <widget class="QDialog" name="NewProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="6">
    <widget class="QLabel" name="lblProjectDir">
     <property name="text">
      <string>Project Directory</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="projectDir"/>
   </item>
   <item row="1" column="1" colspan="5">
    <widget class="QPushButton" name="browseProjectDir">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="QPushButton" name="browseInputVideo">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="inputVideo"/>
   </item>
   <item row="2" column="6">
    <widget class="QLabel" name="lblInputVideo">
     <property name="text">
      <string>Input video</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="7">
    <widget class="QPlainTextEdit" name="txtVideoInfo"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblVideoInfo">
     <property name="text">
      <string>Video information:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblDesc">
     <property name="text">
      <string>Chose project settings:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="7">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bAbort">
       <property name="text">
        <string>Abort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bOk">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
